awx/api/serializers.py

```diff
@@ -817,6 +821,16 @@ def to_representation(self, obj):
         ret['job_explanation'] = _(obj.job_explanation)
         return ret
 
+    def get_launched_by(self,obj):
+        if obj is not None:
+            return obj.launched_by
+    
+    def get_ancestor_job(self,obj):
+        ancestor_job = obj.ancestor_job
+        if ancestor_job is not None:
+            return model_to_dict(ancestor_job)
```

Comment:
@marshmalien can you provide some input on the format of data you're looking for here?

Do you *just* need a URL?

Reply:
@amolgautam25 @jbradberry we discussed this privately, but I'll repeat.

I think this makes more sense as a summary field, not an object-level field.  We should edit `UnifiedJobSerializer.get_summary_fields()` to do something like this, I think:

```python
if self.is_detail_view and obj.ancestor_job != obj:
    ancestor = obj.ancestor_job
    summary_fields['ancestor_job'] = {
        'id' : ancestor.id,
        'name': ...
        'type': ...
        'url': ...
    }
```

Reply:
With the summary field added, we shouldn't need this method anymore.

Reply:
I have removed it.